Mage.Sets/src/mage/cards/g/GoblinPsychopath.java

```diff
@@ -68,6 +74,9 @@ public void init(Ability source, Game game) {
         }
 
         this.wonFlip = controller.flipCoin(source, game, true);
+        game.informPlayers("The next time " + game.getObject(source.getSourceId()).getLogName()
```

Comment:
shouldn't this only inform if the flip is lost?

(probably could be refactored to FlipCoinEffect anyway)

Reply:
Yes. Let me fix